models/ed/R/met2model.ED2.R

```diff
@@ -30,7 +30,7 @@
 ##' @param lst timezone offset to GMT in hours
 ##' @param overwrite should existing files be overwritten
 ##' @param verbose should the function be very verbose
-met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst=0,..., overwrite=FALSE,verbose=FALSE){
+met2model.ED2 <- function(in.path,in.prefix,outfolder,start_date, end_date, lst=0,lat,lon,..., overwrite=FALSE,verbose=FALSE){
```

Comment:
Good point - I forgot about the header.

The nc file has the lat,lon of the actual extraction point which is the closest NARR point to the site lat/lon this could (and most likely will always be different) than the lat,lon of the study site. We can't have the nc lat,lon in the ed met header file, because there's the possibility that it can be out of the polygon. Ask @apourmok if you have more questions.